rdkit/Chem/AllChem.py

```diff
@@ -260,10 +261,14 @@ def _combiEnumerator(items, depth=0):
       else:
         yield [item]
 
+  ProductEduct = namedtuple('ProductEduct', 'products,educts')
```

Comment:
I don't think we should use "educt" here. The rest of the RDKit (at least most of it) uses "reactant". I'd suggest using that in order to remain consistent.